### ISSUE TITLE
Can't find template with search

### ISSUE THREAD

**Issue by ssnn-airr** [metadatacenter/cedar-project#667]
I just created a user in Cedar. I have to upload AIRR data. The search engine doesn't find the AIRR template if I type 'MiAIRR', 'miairr'...

**Comment by ssnn-airr** [comment]
I just created a user in Cedar. I have to upload AIRR data. The search engine doesn't find the AIRR template if I type 'MiAIRR', 'miairr'...

**Comment by graybeal** [comment]
I assume you did find it, is there an issue still that we need to fix?

________________________________

**Comment by ssnn-airr** [comment]
I found it because @ahmadchan helpme me. But other users may not have his help.

**Comment by martinjoconnor** [comment]
What is underlying issue here?

**Comment by graybeal** [comment]
I'm not sure, I think we need more help from @ahmadchan and/or @ssnn-airr about what happened.  I did the following on the staging system—I assume that's what Susanna is using.

I see there is a MiAIRR template (ID bd810a2a-fed0-42a0-9d02-4518f1ea3c1e) that is shared with the AIRR community. Ahmad, is that the one that the AIRR community (read: Susanna) is told to use? If not, what template is the right one?

If Susanna were not a member of the AIRR community when she was searching for that template, she would not have found it. Ahmad, if that *is* the right template, was she a member of the AIRR community?  If she was not a member, is that something we need to add to the CAIRR process documentation? Or do we need to make that template visible to everyone?

(If she was a member and still could not see it, I don't have any idea what the problem could have been, or what could have happened in the meantime so that now she can see it. But I don't think that's the situation, I think the answer lies elsewhere.)

If we can't track this down after getting additional information from Ahmad (and/or Susanna), then we may need to sign this off, and look for more information in a future case.

**Comment by ssnn-airr** [comment]
What I did is what any new user would do, I guess. I created a new user account, I don't know to what user group I belong to. But based on the messages above, maybe my user is not in the Airr community group. 
As a new user, I started with an empty workspace. So, to start working on my AIRR submission, I needed to find the right template. I just wrote "MiAIRR", "miairr", "AIRR",... in the search box and I didn't get any results. The search was looking on my workspace only. Later, when I used the link @ahmadchan  sent to me to the template ID 5b64cd93-fbde-41b8-837e-0d1965e651ce to save a copy in my workspace, the search did work and would find the template that I had just saved in my workspace.
So, my problem was, as a new user, was I was not able to find the template I needed. How/if to address this, I don't know. But maybe the search engine should be able to look for files in the shared folder for all users, even new users?
